### PR TITLE
Improve MDN microformats page

### PR DESCRIPTION
**Pull request by capjamesg**
#### Summary
This PR adds a section listing use cases of microformats, three rel= microformat property examples with interoperable and/or wide use, and makes minor changes to improve the accuracy and readability of the page.

#### Motivation
I am making this change so that someone reading up on microformats on MDN can easily understand what microformats are used for and how microformats might be able to assist the reader in their goals.

#### Related issues
N/A

#### Metadata

- [ ] Adds a new document
- [X] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other erro

### CODE REVIEW

File: files/en-us/web/html/microformats/index.md [hunk 1]
```diff
@@ -12,13 +12,13 @@ tags:
...
+[_Microformats_](https://microformats.org/)are standards used to embed semantics & structured data in HTML, and provide an API to be used by social web applications, search engines, aggregators, and other tools. These minimal patterns of HTML are used for marking up entities that range from fundamental to domain-specific information, such as people, organizations, events, and locations.
 
 - To create a microformats object, h-\* class names are used in the class attribute.
 - To add a property to an object, the p-\*, u-\*, dt-\*, e-\* class names are used on one of the object's descendants.
 
+Microformats use supporting vocabularies to describe objects and name-value pairs to assign values to their properties. The properties are carried in class attributes that can be added to any HTML element, while the data values re-use HTML element content and semantic attributes. Microformats2 (sometimes referred to as mf2) is an update to microformats that provides a simpler way of annotating HTML structured syntax and vocabularies than previous approaches of using RDFa and microdata. These previous approaches require learning new attributes.
```
Comment (ericwbailey) [inline files/en-us/web/html/microformats/index.md:20]:
```suggestion
Microformats use supporting vocabularies to describe objects and name-value pairs to assign values to their properties. The properties are carried in class attributes that can be added to any HTML element, while the data values re-use HTML element content and semantic attributes. 

Microformats2 (sometimes referred to as mf2) is an update to microformats that provides a simpler way of annotating HTML structured syntax and vocabularies than previous approaches of using RDFa and microdata. These previous approaches require learning new attributes.
```

File: files/en-us/web/html/microformats/index.md [hunk 2]
```diff
@@ -31,10 +31,20 @@ An author of a webpage can add microformats to their HTML. For example if they w
...
+When a parser encounters this data, it will know that this page contains a "card" which describes a person or organization named `Alice Blogger` with a URL of `https://alice.example.com/`. The parser makes this data available via APIs that can be used for different applications. For example, an application could scan a page for a h-card to use as profile information for someone who has signed up to a service.
 
 As in this example, some markup patterns require only a single microformat root class name, which parsers use to find a few generic properties such as `name`, `url`, `photo`.
 
+## Microformats Use Cases
+
+Microformats have numerous use cases. First, the Webmention standard uses microformats to provide a way in which messages and comments can be sent from one site to another. The Webmention specification defines specific attributes that sites may publish and consume to create a rich, interoperable way of publishing messages and comments. Microformats can also be used with Webmentions to enable sending social reactions such as likes, reposts, and bookmarks from one site to another.
```
Comment (ericwbailey) [inline files/en-us/web/html/microformats/index.md:40]: Can we link to the Webmention standard?

**Review by ericwbailey — CHANGES_REQUESTED** [review]
As a microformats fan, I really enjoyed your updates. One request and two little tweaks, and I think this will be ready to merge!

**Comment by capjamesg** [comment]
@ericwbailey Thank you for your review! I am also a big microformats fan so am excited about keeping the documentation up to date and relevant. I have made changes per your suggestions in my last two commits.

If there are any other changes or additions to MDN that relate to microformats you think could be made, do let me know. I'd be happy to give those changes a go!

**Review by ericwbailey — APPROVED** [review]

**Comment by ericwbailey** [comment]
@capjamesg Apologies for the delay in getting back to you. Thank you for making the changes, merged! And congratulations on your first contribution to the MDN 🎉

**Pull request merged**